core/tests/protractor.conf.js

```diff
@@ -242,6 +242,7 @@ exports.config = {
       // this thread: https://github.com/angular/protractor/issues/5274
       w3c: false,
       args: [
+        '--headless',
```

Comment:
Is the e2e team ok with this? Seems like the sort of change that should be run past them. /cc @kevintab95 @nithusha21 @U8NWXD @DubeySandeep

Reply:
Oh, this is unintentional. This was just for testing locally, I've reverted it now.